Dockerfile

```diff
@@ -25,7 +26,7 @@ COPY go.mod .
 RUN export GOOS=$TARGETOS && \
     export GOARCH=$TARGETARCH && \
     GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3); export GOARM=${GOARM:1} && \
-    go build -mod vendor -a -o manager main.go
+    go build -mod vendor -a -ldflags "${LDFLAGS:--X github.com/open-policy-agent/gatekeeper/pkg/version.Version=latest}" -o manager main.go
```

Comment:
Does appending `Version=latest` override the default specified in the Makefile?

```
LDFLAGS := "-X github.com/open-policy-agent/gatekeeper/pkg/version.Version=$(VERSION) \
```

Reply:
It would only use latest if LDFLAGS isn't passed in as an arg to the dockerfile.

Reply:
Ah, duh, BASH vars, makes sense.